JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -36,6 +36,7 @@ public class SqsNotificationListener : INotificationSubscriber
             IMessageSerializationRegister serializationRegister,
             IMessageMonitor messagingMonitor,
             ILoggerFactory loggerFactory,
+            IMessageContextAccessor messageContextAccessor,
```

Comment:
This is probably because it's wip, but this class isn't being resolved by DI yet

Reply:
yep

Reply:
Who needs the whole LiveShare coding when you can get feedback commit by commit in real-time!